user/travis-pro.md

```diff
@@ -61,3 +61,16 @@ Travis CI is, and always will be, free for open source projects.
 
 For a list of plans and prices for private repositories, look at
 [travis-ci.com/plans](https://travis-ci.com/plans).
+
+## Making a private repository public
+
+If you have a private repository that you'd like to make public, first change
+the setting on GitHub, then resync your Travis CI account:
+
+1. Follow the [instructions on
+GitHub](https://help.github.com/articles/making-a-private-repository-public/) on
```

Comment:
Sorry for the confusion, @plaindocs - I meant toggling off on Travis CI. 

To recap, if you were to make a private repo open source you would: 

1. Toggle off in Travis Pro travis-ci.com/profile/owner_name 
2. Log in travis-ci.org
3. If the repo does not appear yet in your open source profile, trigger a sync
4. Toggle on the repository in your open source profile page travis-ci.org/profile/owner_name 

Does that make more sense? Thank you!